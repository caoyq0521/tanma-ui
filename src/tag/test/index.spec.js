import { mount } from '@vue/test-utils';
import TmTag from '../tag.vue';

describe('tmTag', () => {
  it('render tag', () => {
    const wrapper = mount('TmTag');
    expect(wrapper).toMatchSnapshot();
  });
})
  