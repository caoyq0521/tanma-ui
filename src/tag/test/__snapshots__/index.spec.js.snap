// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tmTag render tag 1`] = `<span class="tm-tag"><!----> 提示 <!----></span>`;
